### PR TITLE
Add the identity map client docs for python sdk

### PR DESCRIPTION
**Pull request by caroline-ttd**

### CODE REVIEW

File: docs/sdks/uid2-sdk-ref-python.md [hunk 1]
```diff
@@ -214,6 +215,35 @@ If you're using server-only integration (see [Publisher Integration Guide, Serve
 
    If the user has opted out, this method returns `None`, indicating that the user's identity should be removed from the session. To confirm optout, you can use the `token_refresh_response.is_optout()` function.
 
+## Usage for Advertisers and Data Providers
+1. Create an instance of IdentityMapClient as an instance variable.
+   ```py
+   client = IdentityMapClient(base_url, auth_key, secret_key)
+   ```
+
+2. Call a function that takes email addresses or phone numbers as input and generates an IdentityMapResponse object. The following example uses email addresses:
```
Comment (genwhittTTD) [inline docs/sdks/uid2-sdk-ref-python.md:224]: code tagging re response object:
`IdentityMapResponse` object

File: docs/sdks/uid2-sdk-ref-python.md [hunk 2]
```diff
@@ -67,11 +67,12 @@ The package is published in this location:
...
 
-| Role	     | Create Instance of Class	 | Link to Instructions                          |
-|:----------|:--------------------------|:----------------------------------------------|
-| Publisher | `Uid2PublisherClient`     | [Usage for Publishers](#usage-for-publishers) |
-| DSP       | `BidstreamClient`         | [Usage for DSPs](#usage-for-dsps)             |
-| Sharer    | `SharingClient`           | [Usage for Sharers](#usage-for-uid2-sharers)  |
+| Role	                          | Create Instance of Class	 | Link to Instructions                          |
+|:-------------------------------|:--------------------------|:----------------------------------------------|
+| Publisher                      | `Uid2PublisherClient`     | [Usage for Publishers](#usage-for-publishers) |
+| Advertisers and Data Providers | `IdentityMapClient`       | [Usage for Advertisers and Data Providers](#usage-for-advertisers-and-data-providers)     |
```
Comment (genwhittTTD) [inline docs/sdks/uid2-sdk-ref-python.md:73]: Other roles in the table are singular (publisher, DSP, Sharer) so this role should be also (Advertiser/Data Provider).
Note: If you copied this from somewhere else LMK and we can make them consistent. Either is really fine but we have to be consistent.

**Review by genwhittTTD — CHANGES_REQUESTED** [review]
Looks great. Left 3 comments.

File: docs/sdks/uid2-sdk-ref-python.md [hunk 3]
```diff
@@ -214,6 +215,35 @@ If you're using server-only integration (see [Publisher Integration Guide, Serve
 
    If the user has opted out, this method returns `None`, indicating that the user's identity should be removed from the session. To confirm optout, you can use the `token_refresh_response.is_optout()` function.
 
+## Usage for Advertisers and Data Providers
+1. Create an instance of IdentityMapClient as an instance variable.
```
[the remark this answers is not in the document]
Comment (caroline-ttd) [reply docs/sdks/uid2-sdk-ref-python.md]: Added the code tagging

File: docs/sdks/uid2-sdk-ref-python.md (hunk 2 above)
Comment (caroline-ttd) [reply docs/sdks/uid2-sdk-ref-python.md]: It is also used in the java doc, I updated python and java doc both to Advertiser/Data Provider.

**Review by genwhittTTD — APPROVED** [review]
LGTM, thanks!

**Review by asloobq — COMMENTED** [review]
I discussed this recently with Gen in context of EUID docs. The Java SDK doc doesn't mention the functionality of mapping DII to raw UID in the "Functionality" section. Gen added it to EUID docs https://euid.eu/docs/sdks/sdk-ref-java#functionality May be good to include that too

File: docs/sdks/uid2-sdk-ref-python.md [hunk 4]
```diff
@@ -214,6 +215,35 @@ If you're using server-only integration (see [Publisher Integration Guide, Serve
 
    If the user has opted out, this method returns `None`, indicating that the user's identity should be removed from the session. To confirm optout, you can use the `token_refresh_response.is_optout()` function.
 
+## Usage for Advertisers/Data Providers
+1. Create an instance of `IdentityMapClient` as an instance variable.
+   ```py
+   client = IdentityMapClient(base_url, api_key, client_secret)
+   ```
+
+2. Call a function that takes email addresses or phone numbers as input and generates an `IdentityMapResponse` object. The following example uses email addresses:
```
[the remark this answers is not in the document]
Comment (caroline-ttd) [reply docs/sdks/uid2-sdk-ref-python.md:224]: Talked with Jonathan, it is ok we do not include hashed type as input in the guidance. Java doc also does not mention it. https://unifiedid.com/docs/sdks/uid2-sdk-ref-java#usage-for-advertisers-and-data-providers

**Comment by caroline-ttd** [comment]
> I discussed this recently with Gen in context of EUID docs. The Java SDK doc doesn't mention the functionality of mapping DII to raw UID in the "Functionality" section. Gen added it to EUID docs https://euid.eu/docs/sdks/sdk-ref-java#functionality May be good to include that too

Yeah, that is a good point, I added it

File: docs/sdks/uid2-sdk-ref-python.md [hunk 5]
```diff
@@ -38,9 +38,9 @@ You can use the UID2 SDK for Python on the server side to facilitate the followi
 
 This SDK simplifies integration with UID2 for any DSPs or UID2 sharers who are using Python for their server-side coding. The following table shows the functions it supports.
 
-| Encrypt Raw UID2 to UID2 Token | Decrypt UID2 Token | Generate UID2 Token from DII | Refresh UID2 Token |
-| :--- | :--- | :--- | :--- |
-| Supported | Supported | Supported | Supported |
+| Encrypt Raw UID2 to UID2 Token | Decrypt UID2 Token | Generate UID2 Token from DII | Refresh UID2 Token | Map Personal Data to a Raw UID2 |
```
Comment (asloobq) [inline docs/sdks/uid2-sdk-ref-python.md:41]: In UID2 "Personal Data" is called DII. @genwhittTTD you can correct if I am mistaken here

Comment (caroline-ttd) [reply docs/sdks/uid2-sdk-ref-python.md]: You are right, I checked UID2 java docs, it called `DII`. I updated it to `Map DII to a Raw UID2`

**Review by genwhittTTD — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**